pkg/codegen/docs/gen.go

```diff
@@ -856,11 +857,27 @@ func (mod *modContext) getProperties(properties []*schema.Property, lang string,
 			propTypes = append(propTypes, mod.typeString(prop.Type, lang, characteristics, true))
 		}
 
+		comment := prop.Comment
+		// Default values for Provider inputs correspond to environment variables, so add that info to the docs.
+		if isProvider && input && prop.DefaultValue != nil && len(prop.DefaultValue.Environment) > 0 {
+			suffix := ""
```

Comment:
Nit: I personally think this fine, but others might say it's more idiomatic go to declare it as:

```suggestion
			var suffix string
```